### DISCUSSION TITLE
Automatically create a TOC page

### DISCUSSION THREAD

**Discussion by petervanderdoes** [requarks/wiki#5901] [Help / Questions] [answered]
Is it possible to create a page that will hold a TOC linked to other pages in the tree?

Example: I have a structure like this
```
TIL
    javascript
       check-if-a-number-is-positive-or-negative
       configure-jest-to-run-a-test-setup-file
    PostgreSQL
        add-foreign-key-constraint-without-a-full-lock
```

I would like to be able to have a page that will go over this structure and create a page with links automatically
The page can live under TIL

**Comment by NGPixel** [comment, marked as answer]
This is coming in v3. It's not possible at the moment.
